components/adblock_rust_ffi/src/lib.rs

```diff
@@ -373,3 +373,20 @@ pub unsafe extern "C" fn engine_hidden_class_id_selectors(
         .expect("Error: CString::new()")
         .into_raw()
 }
+
+#[cfg(feature = "ios")]
+#[no_mangle]
+pub unsafe extern "C" fn convert_rules_to_content_blocking(rules: *const c_char) -> *mut c_char {
```

Comment:
on the cpp side can we expose this through adblock::FilterList and create adblock::IOSContentRuleList or something? I'd really like to avoid exposing things as functions like this and also it won't require any changes when we convert to cxx

Reply:
I'm not sure what the equivalent API would be, considering that `FilterSet` is taken by ownership rather than reference. I can handle CXX once I can get it to generate some APIs, but I think that's best handled as a followup.